WordPressKit/AccountServiceRemote.h

```diff
@@ -14,13 +14,15 @@ typedef NS_ERROR_ENUM(AccountServiceRemoteErrorDomain, AccountServiceRemoteError
 @protocol AccountServiceRemote <NSObject>
 
 /**
- *  @brief      Gets all blogs for an account.
+ *  @brief      Gets blogs for an account.
  *
- *  @param      success     The block that will be executed on success.  Can be nil.
- *  @param      failure     The block that will be executed on failure.  Can be nil.
+ *  @param      filterJetpackSites    Whether we're fetching only Jetpack blogs.
+ *  @param      success      The block that will be executed on success.  Can be nil.
+ *  @param      failure      The block that will be executed on failure.  Can be nil.
  */
-- (void)getBlogsWithSuccess:(void (^)(NSArray *blogs))success
-                    failure:(void (^)(NSError *error))failure;
+- (void)getBlogs:(BOOL)filterJetpackSites
```

Comment:
@emilylaguna Fixed! 937795a 

I was going back and forth between adding a `getJetpackBlogsWithSuccess:failure:` method OR modifying the existing method... I think your suggestion of keeping the existing method AND adding a new method with the filter flag seems like the best way to avoid regressions 👍